test/test_integration.py

```diff
@@ -417,8 +417,8 @@ def socket_handler(listener):
             # We should get two packets: one connection header string, one
             # SettingsFrame. Rather than respond to the packets, send a GOAWAY
             # frame with error code 0 indicating clean shutdown.
-            first = sock.recv(65535)
-            second = sock.recv(65535)
+            first = sock.recv(65535)  # pragma: no flakes
+            second = sock.recv(65535)  # pragma: no flakes
```

Comment:
Their use was only to do with clarity, so that I know what I'm doing in those tests.